force-app/main/default/lwc/pageSettingsForRelatedCandidatesBlock/pageSettingsForRelatedCandidatesBlock.js

```diff
@@ -0,0 +1,126 @@
+import { LightningElement, wire } from "lwc";
+import getAllQualifiedApiNamesForRelatedCandidate from "@salesforce/apex/PageSettingsRelatedCandidatesController.getAllQualifiedApiNamesForRelatedCandidate";
+import getCustomMetadataForRelatedCandidate from "@salesforce/apex/PageSettingsRelatedCandidatesController.getCustomMetadataForRelatedCandidate";
+import Select_Metadata_Record from "@salesforce/label/c.Select_Metadata_Record";
+import Custom_Metadata_Type_Record from "@salesforce/label/c.Custom_Metadata_Type_Record";
+import Field_Set_For_Candidate_Tile from "@salesforce/label/c.Field_Set_For_Candidate_Tile";
+import Field_Set_For_Candidate_Modal from "@salesforce/label/c.Field_Set_For_Candidate_Modal";
+import Field_Set_For_Job_Application from "@salesforce/label/c.Field_Set_For_Job_Application";
+import Candidate_Per_Page from "@salesforce/label/c.Candidate_Per_Page";
+import Is_Avatars_Shown from "@salesforce/label/c.Is_Avatars_Shown";
+import Is_Unaccessable_Shown from "@salesforce/label/c.Is_Unaccessable_Shown";
+import Msg_Record_Update from "@salesforce/label/c.Msg_Record_Update";
+import Fields from "@salesforce/label/c.Fields";
+import Save from "@salesforce/label/c.Save";
```

Comment:
I belive a bit better to use _label instead because save is more about action.
```suggestion
import Save_Label from "@salesforce/label/c.Save";
```